.circleci/config.yml

```diff
@@ -1,145 +1,376 @@
 version: 2.1
 
 orbs:
-  gem: doximity/gem-publisher@0
```

Comment:
Removed gem publishing. See https://github.com/doximity/works-on-my-machine/tree/master for our new way to handle open source gem releases